File: ui/android/wuffittracker/tests/src/com/androidworks/navsys/wuffit/WuffITApplicationTest.java
package com.androidworks.navsys.wuffit;

import android.test.ApplicationTestCase;

public class WuffITApplicationTest extends ApplicationTestCase<WuffITApplication> {
    public WuffITApplicationTest() {
        super(WuffITApplication.class);
    }

    @Override
    protected void setUp() throws Exception {
        super.setUp();
        createApplication();
    }
}

File: ui/android/wuffittracker/src/androidTest/kotlin/com/androidworks/navsys/wuffit/WuffITApplicationTest.kt
package com.androidworks.navsys.wuffit

import androidx.test.ext.junit.runners.AndroidJUnit4
import androidx.test.platform.app.InstrumentationRegistry
import org.junit.Assert.assertNotNull
import org.junit.Test
import org.junit.runner.RunWith

@RunWith(AndroidJUnit4::class)
class WuffITApplicationTest {

    @Test
    fun testAppContext() {
        val appContext = InstrumentationRegistry.getInstrumentation().targetContext
        assertNotNull(appContext)
    }
}
